Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -630,6 +630,9 @@ protected Vector<String> handleDeepLinkIfAvailable(Intent appLinkIntent){
                 if (key.equals("vp")){
                     viewParam = query_params.get(key);
                     break;
+                } else if (key.equals("referrer")) {
```

Comment:
we should use smaller query params to keep the length of links shorter. Like rf here for referrer/referral

Reply:
yes, but in this case we are using similar link for playStore referral and deepLink where playStore accepts query as `referrer`